subprojects/architecture-test/src/changes/archunit_store/internal-api-nullability.txt

```diff
@@ -6584,4 +6584,4 @@ Class <org.gradle.wrapper.PathAssembler$LocalDistribution> is not annotated (dir
 Class <org.gradle.wrapper.PathAssembler> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (PathAssembler.java:0)
```

Comment:
These files here are auto-generated. So we'd need to fix the generation and not the files themselves.